### PR TITLE
Fixes to Token Deploy dapp

### LINKED ISSUE #4311: Token Deployment ignores 0 decimals entered
... while always defaulting to 6 decimals. Either -

1. Decimals (if fixed should be explicitly stated)
2. If decimals overrides allowed, send proper values

### PR DESCRIPTION
**Pull request by ngotchac**
Fixes #4311 

Now can use decimals in the token deploy.
Several UI fixes to the dapp.

### CODE REVIEW

File: js/src/dapps/tokendeploy/Transfer/Send/send.js [hunk 1]
```diff
@@ -121,8 +121,21 @@ export default class Send extends Component {
   }
 
   renderForm () {
+    const { tokens } = this.state;
+
+    if (!tokens || tokens.length === 0) {
+      return (
+        <Container>
+          <div className={ styles.statusHeader }>
+            There are tokens to transfer
```
Comment (jacogr) [inline js/src/dapps/tokendeploy/Transfer/Send/send.js]: There are no tokens to transfer?

Comment (ngotchac) [inline js/src/dapps/tokendeploy/Transfer/Send/send.js]: Fixed

**Comment by jacogr** [comment]
I just need to play a bit - no issues found at first run-through, but need to get my head around this with some playing.

**Comment by jacogr** [comment]
Just one small visual issue with the new longer errors (input jumps down for me when error occurs) -

jump-error

**Comment by ngotchac** [comment]
Updated !

**Pull request merged**